### PR TITLE
increase nav bar custom icons tap area

### PR DESCRIPTION
**Pull request by Cool04ek**

### CODE REVIEW

File: ios/RCCNavigationController.m [hunk 1]
```diff
@@ -406,13 +406,16 @@ -(void)setButtons:(NSArray*)buttons viewController:(UIViewController*)viewContro
     }
   }
   
+  UIBarButtonItem *itemToIncreaseInsets = [barButtonItems lastObject];
```
Comment (eliperkins) [inline ios/RCCNavigationController.m]: Why does this call `lastObject`? I don't understand why we only need to change insets for the last item.

Comment (Cool04ek) [reply ios/RCCNavigationController.m]: In other case the previous ones insets may overlap with the next ones

**Comment by eliperkins** [comment]
I'm not sure I understand why this change is needed. Can you elaborate more as to what bug this is fixing?

**Comment by Cool04ek** [comment]
It's not a bug fix. It's feature, asked by product manager.

**Comment by eliperkins** [comment]
This doesn't seem like much of a feature if it's something that will affect all consumers of this library, regardless of product. It would be great if we could figure out what the feature is that you're trying to add in and find a way to get it in that is backwards compatible for other users.

**Comment by Cool04ek** [comment]
Basically, it increases tap area of buttons. It's that simple

**Comment by guyca** [comment]
@Cool04ek Can we make this change opt in?

**Pull request closed without merging**